openrouteservice/src/main/java/heigit/ors/routing/graphhopper/extensions/flagencoders/ORSAbstractFlagEncoder.java

```diff
@@ -35,6 +36,10 @@ public void setConsiderElevation(boolean considerElevation) {
 		this.considerElevation = considerElevation;
 	}
 
+	public double getMeanSpeed() {
```

Comment:
It would be better to define this as an abstract method that must be overridden by the sub classes as then it ensures that we will always have something that is meant to be returned. currently if someone was to use the ORSAbstractFlagEncoder class without overriding, the mean speed would be 0 (the default value for a double).

Reply:
jup, thx for the instructions:)